modules/@apostrophecms/permission/index.js

```diff
@@ -288,7 +282,15 @@ module.exports = {
       },
       matchTypicalPieceType(permissionSet) {
```

Comment:
What is a typical piece type?

Reply:
I don't think this is requiring all custom doc types to include `showPermissions: true`, but I also am not fully understanding how a normal dev should think about that. And what purpose it's serving other than excluding when we say `false`. Right now it sounds like it's a binary choice, but it doesn't seem like it actually is.

Reply:
The design for the permissions explorer calls for folding all "typical, ordinary" pieces into one section, unless they (a) have permissions that actually would not match what is shown there, (b) explicitly opt into having their own entry in the permissions explorer with `showPermissions: true`, or (c) explicitly opt out of being mentioned at all in the permissions explorer, even in the "typical" section's tooltip, with `showPermissions: false`.

This is not really new. All that is new is that I removed the `interestingPieces` option, which would have been difficult to extend and maintain, in favor of a setting that can be made on a per-type level so that new types can specify their treatment in the permissions explorer.

Reply:
Got it. I think the method naming could be clearer, but I think the actual option makes sense.